instagram/oauth2.py

```diff
@@ -109,7 +109,8 @@ def exchange_for_access_token(self, code=None, username=None, password=None, sco
         data = self._data_for_exchange(code, username, password, scope=scope, user_id=user_id)
         http_object = Http(disable_ssl_certificate_validation=True)
         url = self.api.access_token_url
-        response, content = http_object.request(url, method="POST", body=data)
+        headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+        response, content = http_object.request(url, method="POST", body=data, headers=headers)
```

Comment:
It'd be great it this made it into the official library, this saved me just now. Thanks @vgavro!